src/testing/run.ts

```diff
@@ -158,6 +159,11 @@ async function runTestCaseUnsafe<TestCaseType, OutputType>(args: {
     },
   );
 
+  // Flush the logs before we send the result, since the CLI
+  // accumulates the events and sends them as a batch along
```

Comment:
'the events' refers to test specific events right?

Reply:
yeah, when in a testing context we send the events to `<cli-address>/events`, and in the CLI it's just adding them to an array and then sending the events it's seen so far when we finally POST to `<cli>/results`:

https://github.com/autoblocksai/cli/blob/fa5d09dea853c019fe55e1447a06b1ccded39e4a/src/handlers/testing/exec/index.ts#L211-L222

https://github.com/autoblocksai/cli/blob/fa5d09dea853c019fe55e1447a06b1ccded39e4a/src/handlers/testing/exec/index.ts#L243-L265

so we need to make sure all the events have been flushed before we make that POST request to `/results`